### DISCUSSION TITLE
切换应用后回来发现TikTok被杀后台，重开就无法联网了

### DISCUSSION THREAD

**Discussion by nivranic** [Semporia/TikTok-Unlock#280] [Q&A]
v一直开着

**Comment by nivranic** [comment]
从21.1.0抓包，然后升级，升级后第一次点开也能用，但后来就如上所说了，搞不懂

**Reply by nivranic** [reply]
没登录账号关后台重开不受影响，登了账号就成了一次性的。另外我时区语言地区都没改

**Reply by Ledrong** [reply]
你好，我是來自俄羅斯的學生。我想在https://buff.163.com/平台進行交易。但為此我需要驗證我的帳戶，而這只能使用支付寶和steam來完成。請您只需要驗證一下，一切都是安全的。我會付1000元
